lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -236,7 +238,7 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
     }
 
     async getFolderTask(appName: string, taskId: string) {
-        this.nodeId = await this.formCloudService.getProcessStorageFolderTask(appName, taskId).toPromise();
+        this.processStorageCloudModel = await this.formCloudService.getProcessStorageFolderTask(appName, taskId).toPromise();
```

Comment:
the method is called "getFolderTask" but it does not return anything

Reply:
Thanks @DenysVuika  I will fix it